.travis.yml

```diff
@@ -19,4 +19,4 @@ before_install:
   - tar -xzf ../apache-cassandra-2.1.12-bin.tar.gz -C ../
   - sh ../apache-cassandra-2.1.12/bin/cassandra > /dev/null
 
-script: sh test/utils/run_tests.sh coverage all && (npm run-script coveralls || exit 0)
+script: (sh test/utils/run_tests.sh coverage all && npm run-script coveralls || exit 0)
```

Comment:
Em, what're we trying to achieve here? 

I thought the previous behaviour was 'not to fail on coveralls issue, but fail unconditionally if the tests failed'. Now it seems the exit code would be 0 in any case, even if the tests actually failed. (1 && 1 || 0 === 0)